### ISSUE TITLE
Installer requires admin privileges [feature request]

### ISSUE THREAD

**Issue by sellicott** [cnlohr/tinycc-win64-installer#1]
As it stands, the installer requires admin privileges to run. It would be nice to have a version of the program that can be user installed to the user "home directory". Editing the path probably isn't possible in this case, but I think that is a worthwhile tradeoff.

**Comment by cnlohr** [comment]
I agree! This would be awesome! But, I don't know how to do that. At least as an "option" so to speak.  Maybe a separate installer?
